dask/tests/test_base.py

```diff
@@ -983,6 +983,17 @@ def test_persist_scalar_rename():
     dd.utils.assert_eq(dds2, 5)
 
 
+@pytest.mark.skipif("not dd or not np")
+def test_dataframe_multi_index():
+    # see https://github.com/dask/dask/issues/4845
+    df = pd.DataFrame({"ind_a": np.arange(100), "ind_b": 1, "var": "whatever"})
+    df = dd.from_pandas(df, npartitions=90)
+
+    # Only fails when grouping with two variables.
```

Comment:
I think this comment is likely to be confusing to people who haven't read the initial issue. Maybe something like "in older versions of pandas, this would fail due to a thread-safety issue"